bn_mp_tc_and.c

```diff
@@ -0,0 +1,90 @@
+#include "tommath_private.h"
+#ifdef BN_MP_TC_AND_C
+/* LibTomMath, multiple-precision integer library -- Tom St Denis
+ *
+ * LibTomMath is a library that provides multiple-precision
+ * integer arithmetic as well as number theoretic functionality.
+ *
+ * The library was designed directly after the MPI library by
+ * Michael Fromberger but has been written from scratch with
+ * additional optimizations in place.
+ *
+ * The library is free for all purposes without any express
+ * guarantee it works.
+ */
+
+/* two complement and */
+int mp_tc_and(const mp_int *a, const mp_int *b, mp_int *c)
+{
+   int res = MP_OKAY, bits;
+   int as = mp_isneg(a), bs = mp_isneg(b), s = 0;
+   mp_int *max = 0, _max, acopy, bcopy;
```

Comment:
I don't like the idea to use variable names that could clash with stdlib functions :) [`man bcopy`](https://linux.die.net/man/3/bcopy)

Reply:
fixed